File: Departamento.java
import java.util.ArrayList;

public class Departamento {
    private String nombre;
    private ArrayList<Empleado> empleados;

    public Departamento(String nombre) {
        this.nombre = nombre;
        empleados = new ArrayList<>();
    }

    public String getNombre() {
        return nombre;
    }

    public void agregarEmpleado(Empleado e) {
        empleados.add(e);
    }

    public void listarEmpleados() {
        if (empleados.isEmpty()) {
            System.out.println("No hay empleados.");
            return;
        }

        for (Empleado e : empleados) {
            e.mostrarInfo();
        }
    }

    public void eliminarEmpleado(String nombre) {
        empleados.removeIf(e -> e.getNombre().equalsIgnoreCase(nombre));
    }
}

File: model/departamento.py
class Departamento:
    def __init__(self, nombre):
        self.nombre = nombre
        self.empleados = []

    def agregar_empleado(self, e):
        self.empleados.append(e)

    def listar_empleados(self):
        for e in self.empleados:
            e.mostrar_info()

    def eliminar_empleado(self, nombre):
        self.empleados = [e for e in self.empleados if e.nombre != nombre]
